index.js

```diff
@@ -165,7 +169,11 @@ var Omegle = function () {
 	this.reloadReCAPTCHA = function () {
 		if (challengeLink) evalCaptcha(challengeLink);
 	};
-	this.updateServer = function () {
+	this.updateServer = function (server) {
+		if(server){	//in case user wants to do it manually
+			url='http://'+server;
+			return false;
+		}
```

Comment:
This was required in my case, because only selected servers work for me, maybe country restrictions. front9.omegle.com always refused connections, so whenever it was set as `url` , there is no other way to reconnect except manually setting server.